app/src/main/java/no/usn/mob3000/domain/usecase/auth/RegisterUseCase.kt

```diff
@@ -0,0 +1,27 @@
+package no.usn.mob3000.domain.usecase.auth
+
+import no.usn.mob3000.data.repository.auth.AuthRepository
+import no.usn.mob3000.data.source.remote.auth.AuthDataSource
+import no.usn.mob3000.domain.model.User
+
+/**
+* @property authRepository
+* @author Anarox
+* @created 2024-10-30
+*/
+
+
+class RegisterUseCase(private val authRepository: AuthRepository = AuthRepository(AuthDataSource())) {
```

Comment:
Remember correct spacing. And also remember to document the function. The property needs documentation, and the use case needs documentation. They should not be left blank.